types/config.ts

```diff
@@ -49,5 +49,11 @@ export const ConnectElementCustomMethodConfig = {
     setFetchEphemeralKey: (
       _fetchEphemeralKey: FetchEphemeralKeyFunction | undefined
     ): void => {}
+  },
+  "financial-account": {
+    setFinancialAccount: (_financialAccount: string | undefined): void => {}
```

Comment:
Is the right type string | undefined here? What's the rationale for "undefined"?